docs/aws-s3-sink.md

```diff
@@ -0,0 +1,206 @@
+| `accessKeyId`                   | String  | True     | null         | The AWS access key ID. It requires permission to write objects.                                                                                                                                                                                                                                                                                                                       |
+| `secretAccessKey`               | String  | True     | null         | The AWS secret access key.                                                                                                                                                                                                                                                                                                                                                            |
+| `bucket`                        | String  | True     | null         | The AWS S3 bucket.                                                                                                                                                                                                                                                                                                                                                                    |
+| `formatType`                    | String  | True     | null         | The data format type. Available options are JSON, Avro, Bytes, or Parquet. By default, it is set to JSON.                                                                                                                                                                                                                                                                             |
+| `partitionerType`               | String  | True     | null         | The partitioning type. It can be configured by topic partitions or by time. By default, the partition type is configured by topic partitions.                                                                                                                                                                                                                                         |
+| `region`                        | String  | False    | null         | The AWS S3 region. Either the endpoint or region must be set.                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Do we need to mention or link to how to get the region for the AWS?

Reply:
I don't think it's necessary. This should be common sense.